packages/wrangler/src/pages/dev.tsx

```diff
@@ -385,6 +385,9 @@ export const Handler = async ({
 					define: {
 						__ROUTES__: routes,
 					},
+					inject: [
+						resolve(getBasePath(), "src/pages/functions/routing-rules-util.ts"),
```

Comment:
It's not great to rely on `src` in the distributable. This will work because we do still ship `src` on npm, but I'd rather you move this to `templates` or just add it in the `templates/pages-dev-pipeline.ts` file.

Reply:
@GregBrimble this is addressed now. Lemme know what u think